doc/reference/usermode/syscalls.rst

```diff
@@ -197,17 +170,19 @@ necessary privilege elevation. In this layer, all arguments are treated as an
 unsigned 32-bit type. There is always a 32-bit unsigned return value, which
 may or may not be used.
 
-Some system calls may have more than six arguments. The number of arguments
-passed via registers is fixed at six for all architectures. Additional
-arguments will need to be passed in a struct, which needs to be treated as
-untrusted memory in the handler function. This is done by the derived
-functions :c:func:`_syscall_invoke7` through :c:func:`_syscall_invoke10`.
+Some system calls may have more than six arguments. The number of
+arguments passed via registers is fixed at six for all
```

Comment:
Is it ``fixed at six`` (meaning all system calls pass six arguments) or should it say ``limited to six``?